openshift/ups-template.json

```diff
@@ -0,0 +1,529 @@
+            }
+          },
+          {
+            "type": "ConfigChange"
+          }
+        ],
+        "replicas": 1,
+        "selector": {
+          "name": "keycloak"
+        },
+        "template": {
+          "metadata": {
+            "labels": {
+              "name": "keycloak"
+            }
+          },
+          "spec": {
+            "containers": [
+              {
+                "name": "mysql",
+                "image": " ",
+                "ports": [
+                  {
+                    "containerPort": 3306
+                  }
+                ],
+                "readinessProbe": {
+                  "timeoutSeconds": 1,
+                  "initialDelaySeconds": 5,
+                  "exec": {
+                    "command": [
+                      "/bin/sh",
+                      "-i",
+                      "-c",
+                      "MYSQL_PWD=\"keycloak\" mysql -h 127.0.0.1 -u $MYSQL_USER -D keycloak -e 'SELECT 1'"
+                    ]
+                  }
+                },
+                "livenessProbe": {
+                  "timeoutSeconds": 1,
+                  "initialDelaySeconds": 30,
+                  "tcpSocket": {
+                    "port": 3306
+                  }
+                },
+                "env": [
+                  {
+                    "name": "MYSQL_USER",
+                    "value": "keycloak"
+                  },
+                  {
+                    "name": "MYSQL_PASSWORD",
+                    "value": "keycloak"
+                  },
+                  {
+                    "name": "MYSQL_ROOT_PASSWORD",
+                    "value": "keycloak"
+                  },
+                  {
+                    "name": "MYSQL_DATABASE",
+                    "value": "keycloak"
+                  }
+                ],
+                "resources": {
+                  "limits": {
+                    "memory": "512Mi"
+                  }
+                },
+                "volumeMounts": [
+                  {
+                    "name": "keycloak-data",
+                    "mountPath": "/var/lib/mysql/data"
+                  }
+                ],
+                "imagePullPolicy": "IfNotPresent"
+              }
+            ],
+            "volumes": [
+              {
+                "name": "keycloak-data",
+                "persistentVolumeClaim": {
+                  "claimName": "keycloak"
+                }
+              }
+            ]
+          }
+        }
+      }
+    },
+	
+	{
+	  "apiVersion": "v1",
+	  "kind": "Route",
+	  "metadata": {
+	    "name": "ups-localhost-unsecure"
+	  },
+	  "spec": {
+	    "host": "ups.127.0.0.1.nip.io",
+	    "to": {
+	      "kind": "Service",
+	      "name": "unifiedpush-server",
+	      "weight": 100
+	    },
+	    "port": {
+	      "targetPort": "8080-tcp"
+	    },
+	    "wildcardPolicy": "None"
+	  },
+	  "status": {
+	    "ingress": [
+	      {
+		"host": "ups.127.0.0.1.nip.io",
```

Comment:
when exporting template all status sections should be removed as they aren't applied to new infrastructure.